examples/c_api/vfs.c

```diff
@@ -92,7 +92,8 @@ void dirs_files() {
   tiledb_ctx_free(&ctx);
 }
 
-void write() {
+// Should be static to avoid collision with the write syscall.
+static void write() {
```

Comment:
It would be clearer to rename the functions.

Reply:
Renamed.